backend/src/main/java/com/zeros/backend/Restaurant.java

```diff
@@ -4,18 +4,28 @@
 
 public class Restaurant {
 
+    private Long id;
     private String name;
     private Location location;
     private float rating;
     private List<String> photosReference;
+    private Long mealsCounter;
 
```

Comment:
if we store the scan transactions in the same place as Restaurant information we will coupling 2 deferents domains.
in this way it's will be hard for us to evolve those functionalities separately.
i suggest you to not store meals counts here